scripts/stetho_open.py

```diff
@@ -17,12 +17,12 @@
 import struct
 import re
 
-def stetho_open(device=None, process=None):
-  adb = _connect_to_device(device)
+def stetho_open(device=None, process=None, port=5037):
```

Comment:
port should probably default to `None` and perform the same check you performed in `dumpapp` here as well. This should enable scripts that are using just `stetho_open.py` to actually use the env variable as well